wwwroot/integrations.html

```diff
@@ -0,0 +1,435 @@
+                        <div class="col-12 col-sm-6 col-xl-4 mb-4">
+                            <div class="feature-box">
+                                <img src="img/shield.png" class="icon-size">
+                                <div class="title my-3">Adhere to Privacy Laws</div>
+                                <p>Respect local and regional consumer privacy laws by protecting personal user transaction information.</p>
+                            </div>
+                        </div>
+                        <div class="col-12 col-sm-6 col-xl-4 mb-4">
+                            <div class="feature-box">
+                                <img src="img/feather.png" class="icon-size">
+                                <div class="title my-3">Low Maintenance</div>
+                                <p>Deploy-and-forget maintenance with a backend infrastructure overhead fully managed by zkSNACKs offering a high uptime.</p>
+                            </div>
+                        </div>
+                        <div class="col-12 col-sm-6 col-xl-4 mb-4">
+                            <div class="feature-box">
+                                <img src="img/people.png" class="icon-size">
+                                <div class="title my-3">Reduce Data Breach Risk</div>
+                                <p>Mitigate the impact to your customers in case of data breaches from exchange and brokerage platforms.</p>
+                            </div>
+                        </div>
+                        <div class="col-12 col-sm-6 col-xl-4 mb-4">
+                            <div class="feature-box green">
+                                <div class="floating-content">
+                                    <div class="title my-3">Get in Touch</div>
+                                    <p>If you are interested in learning more about the coinjoin API, please contact us at:
+                                        <br><br>
+                                        <a href="mailto:integrations@zksnacks.com">integrations@zksnacks.com</a>
+                                    </p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <section class="p-4">
+                <div class="container">
+                    <div class="row">
+                        <div class="col-12 col-sm-8 offset-sm-2 text-center">
+                            <div class="my-5">
+                                <h2 class="pt-header title-lined">Eligibility Criteria</h2>
+
+                                <p>Businesses and developers who want their Bitcoin products eligible to integrate the coinjoin API may be asked to meet certain technical requirements around network-level privacy, such as using block filters and Tor.</p>
+
+                                <a href="#" class="btn btn-link">Read more →</a>
```

Comment:
I think this should link somewhere else.
to the blog, like on zksnacks website?

Reply:
Good catch! Fixed.